apollo-backend/src/main/java/io/logz/apollo/kubernetes/KubernetesHandler.java

```diff
@@ -129,6 +129,10 @@ Deployment cancelDeployment(Deployment deployment) {
     }
 
     Deployment monitorDeployment(Deployment deployment) {
+        return monitorDeployment(deployment, Optional.empty());
+    }
+
+    Deployment monitorDeployment(Deployment deployment, Optional<Integer> scalingFactor) {
```

Comment:
Why pass scalingFactor?
You can get the deployed group from the deployment - this is a bit of a smell

Reply:
This is what I was going for at first, but then the KubernetesHandler needs a GroupDao, which is a worse smell..